src/Observers/UserObserver.php

```diff
@@ -5,16 +5,16 @@
 namespace Blumilk\Meetup\Core\Observers;
 
 use Blumilk\Meetup\Core\Models\User;
-use Illuminate\Contracts\Hashing\Hasher;
+use Blumilk\Meetup\Core\Services\PasswordHashingService;
 
 class UserObserver
 {
     public function __construct(
-        protected Hasher $hasher,
+        protected PasswordHashingService $service,
```

Comment:
Ay way. I was thinking about removing hashing entirely from the observer. It should be done in some user registrar.